acis_thermal_check/tests/psmc/answers/FEB1020A_viol.json

```diff
@@ -13,8 +13,8 @@
     ],
     "run_start": "2020:041:00:20:02.553",
     "limits": {
-        "yellow_hi_limit": 51.5,
-        "plan_hi_limit": 47.0
+        "yellow_hi": 51.5,
```

Comment:
Likely these are just for testing purposes, but just flagging that these limits aren't wha the 1PDEAAT limits would have been in Feb 2020.

Reply:
That's correct--I'm purposefully setting these to be lower to force violations.